### DISCUSSION TITLE
use API with a proxy website

### DISCUSSION THREAD

**Discussion by zzzSEA** [nhaouari/obsidian-textgenerator-plugin#217] [Q&A]
Can i use  API with a proxy website? If so, how do i do it?

**Comment by crybest** [comment]
I want this too

**Reply by zzzSEA** [reply]
u can use it i have test

**Reply by lossfunc** [reply]
have you made it work with proxy api? not work for me

**Reply by lossfunc** [reply]
I did it! You need to look at the obsidian web request using cmd+opt+i to make sure the requested URL is correct. I had success by adding /v1 after my base path   <img width="467" alt="image" src="https://github.com/nhaouari/obsidian-textgenerator-plugin/assets/19942192/87f62898-eb99-49cf-bf40-110e32bc84f7">
